packages/@lwc/style-compiler/src/scope-at-rules/transform.ts

```diff
@@ -7,9 +7,25 @@
 import { Root } from 'postcss';
 import { SHADOW_ATTRIBUTE } from '../utils/selectors-scoping';
 
+// Based on a subset from https://github.com/wooorm/vendors/blob/2f489ad/index.js
+// Subset of prefixes for animation-related names that we expect people might be using.
+// The most important is -webkit, which is actually part of the spec now. All -webkit prefixes
+// are listed here: https://developer.mozilla.org/en-US/docs/Web/CSS/Webkit_Extensions
+const VENDOR_PREFIXES = ['moz', 'ms', 'o', 'webkit'];
```

Comment:
We could probably get away with just doing `-webkit` and `-moz` (since `-ms` never seems to have existed for animations, and Opera has been Blink since 2013), but I figure doing all of them will at least make debugging less complicated for developers.